lexer.l

```diff
@@ -22,6 +22,9 @@
 #define LINENO (yylineno - 1)
 %}
 
+/* C99 7.1.3: All identifiers that begin with an underscore and either an uppercase letter or another
```

Comment:
I suggest that we have our line length under 80 characters 😀 
See [Google C++ Style Guide: Line Length](https://google.github.io/styleguide/cppguide.html#Line_Length) for the details!